app/src/main/java/com/nextcloud/talk/adapters/messages/MagicSystemMessageViewHolder.java

```diff
@@ -82,7 +82,9 @@ public void onBind(ChatMessage message) {
                 if (individualMap != null &&
                         ("user".equals(individualMap.get("type")) ||
                                 "guest".equals(individualMap.get("type")) ||
-                                "call".equals(individualMap.get("type"))
+                                "call".equals(individualMap.get("type")) ||
+                                "group".equals(individualMap.get("type")) ||
+                                "circle".equals(individualMap.get("type"))
```

Comment:
they should not be prefixed with an `@` and just be replaced normally (like in the web)